src/libsync/propagateupload.h

```diff
@@ -321,6 +322,99 @@ private slots:
     void slotUploadProgress(qint64,qint64);
 };
 
+/**
+ * @brief The PropagateUploadBundle class is a container class for upload jobs under chunking size.
+ *
+ * It will also ensure proper bandwidth utilization vs bookkeeping balance, and that in case no other items then under chunk uploads are available,
```

Comment:
I don't understand the detailed meaning of:
"and that in case no other items then under chunk uploads are available,"